isis/tests/FunctionalTestsCubeatt.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+
+#include "Cube.h"
+#include "Fixtures.h"
+
+#include "cubeatt.h"
+#include "gmock/gmock.h"
+
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/cubeatt.xml").expanded();
+
+// Tests setting output attributes: bit type and range
+TEST_F(SmallCube, FunctionalTestCubeattBitttypeAndRange) {
+  QString cubePath = "/scratch/without.cub+8bit+0.0:1.0";
+  QVector<QString> args = {"from=" + testCube->fileName(), "to=" + cubePath};
+  UserInterface options(APP_XML, args);
+  cubeatt(options);
+
+  // Test the label
+  Cube outputCube(cubePath);
+  Pvl *label = outputCube.label();
+  PvlGroup &pixelGroup = label->findObject("IsisCube").findObject("Core").findGroup("Pixels");
+  EXPECT_EQ(QString(pixelGroup["Type"]).toStdString(), "UnsignedByte");
+  EXPECT_NEAR(pixelGroup["Base"][0].toDouble(), -0.003952569, 0.00001);
+  EXPECT_NEAR(pixelGroup["Multiplier"][0].toDouble(), 0.003952569, 0.00001);
+
+  // Test the DNs
```

Comment:
I need to add this part of the tests still.

Reply:
I would pull the Cube statistics and check that the min/max values are between the specified values in the cubeatt call. The small cube fixture has values ranging from 0 to 1000, so I would pick something other than 0:1 for your range in this test. Maybe something like 100:460 that will cover about half of the pixels?